sca/windows/win_audit_rcl.yml

```diff
@@ -14,6 +14,13 @@ policy:
   name: "Benchmark for Windows audit"
   description: "This document provides a way of ensuring the security of the Windows systems."
 
+requirements:
+  title: "Check for Windows platform"
+  description: "Requirements for Windows"
```

Comment:
The description can offer more details: `Requirements for running the audit policy under a Windows platform` for example.